packages/kit/src/views/Earn/components/Overview.tsx

```
@@ -0,0 +1,430 @@
+                    </XStack>
+                    <EarnActionIcon
+                      actionIcon={item.button}
+                      onHistory={handleHistoryClick}
+                    />
+                  </>
+                ),
+              });
+            })}
+            {isEmpty(rebateData?.undistributed) ? null : (
+              <SizableText mx="$5" size="$bodyMdMedium" color="$textSubdued">
+                {intl.formatMessage({ id: ETranslations.referral_undistributed })}
```

ESLint

[error] Replace `·id:·ETranslations.referral_undistributed` with `⏎··················id:·ETranslations.referral_undistributed,⏎···············`

(prettier/prettier)